enterprise_subsidy/apps/core/models.py

```diff
@@ -15,6 +16,7 @@ class User(AbstractUser):
 
     """
     full_name = models.CharField(_('Full Name'), max_length=255, blank=True, null=True)
+    history = HistoricalRecords()
```

Comment:
As we discussed, I wasn't sure we actually want this history, especially considering it would need extra work to clear out the PII in the GDPR pipeline, and we don't track User history anywhere else AFAIK.